openstf/pipeline/optimize_hyperparameters.py

```diff
@@ -1,166 +0,0 @@
-# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
Why was this file deleted?

Reply:
This hyperparameter optimization still uses the old models so I decided to delete it on this branch. Optimizing the new sklearn compatible models probably has to be implemented differently. I see there is currently no issue for this but I will make one.